cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1595,166 +1595,280 @@ action.summary_index.alert_name = Splunk Admin - Missing Forwarder
 dispatch.earliest_time = -62m@m
 dispatch.latest_time = -2m@m
 display.general.type = statistics
 display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_tenable_assets` | eval ip=mvappend(ipv4s, ipv6s), tenable_all_ips=mvjoin(ip, ", ") | mvexpand ip | eval hostnames=coalesce(hostnames, fqdns) | fillnull hostnames value="-" | mvexpand hostnames | dedup ip, hostnames \
-| rename host as tenable_collected_by, network_name as tenable_network_name, state as tenable_state | eval host=lower(hostnames) \
-| eval tenable_mac_address=mvjoin(mac_addresses, ", "), tenable_fqdn=mvjoin(fqdns, ", "), tenable_netbios=mvjoin(netbios_names, ", "), tenable_os=mvjoin(operating_systems, ", ") \
-| table _time, tenable_collected_by, ip, host, tenable_all_ips, tenable_mac_address, tenable_fqdn, tenable_netbios, tenable_os, tenable_state, tenable_network_name, created_at, first_scan_time, first_seen, has_agent, has_plugin_results, last_authenticated_scan_date, last_licensed_scan_date, last_seen \
-| append [| inputlookup cs_tenable_assets] \
-| dedup ip, host sortby -_time \
-| outputlookup cs_tenable_assets
-
-[Tenable Vuln Lookup Gen]
-disabled = 1
+search = | sophosinstancedetails all_endpoints=True | extract \
+| rename id as sophos_uuid, "tenant.id" as sophos_customer_id, type as sophos_type, "health.overall" as sophos_health, "health.threats.status" as sophos_threats_status, "health.services.status" as sophos_services_status, "lockdown.status" as sophos_lockdown_status, "lockdown.updateStatus" as sophos_lockdown_update_status, "associatedPerson.name" as sophos_user, "associatedPerson.viaLogin" as sophos_login_via, "os.name" as sophos_os | fillnull "os.build", "os.isServer", "os.majorVersion", "os.minorVersion", "os.platform" value="-" | eval sophos_os_details = mvappend("Platform:".'os.platform', "Build No.: ".'os.build', "ServerOS:".'os.isServer', "OS MajorVersion:".'os.majorVersion', "OS MinorVersion:".'os.minorVersion') \
+| eval service_status=mvzip('health.services.serviceDetails{}.name', 'health.services.serviceDetails{}.status') \
+| eval sophos_product_installed=mvzip('assignedProducts{}.code', mvzip('assignedProducts{}.status', 'assignedProducts{}.version')) \
+| eval _time=strptime(lastSeenAt, "%FT%T.%3QZ"), ip=lower(mvjoin(mvdedup(mvappend('ipv4Addresses{}','ipv6Addresses{}')),"~~")), hostname=lower(hostname), mac_address=lower(mvjoin('macAddresses{}',"~~")) \
+| table _time, sophos_customer_id, sophos_uuid, ip, hostname, mac_address, sophos_os, sophos_os_details, sophos_user, sophos_login_via, tamperProtectionEnabled, service_status, sophos_product_installed, sophos_health, sophos_services_status, sophos_threats_status, sophos_lockdown_status, sophos_lockdown_update_status, sophos_type \
+| append [| inputlookup cs_sophos_inventory | rename time as _time] \
```

Comment:
Same as Lansweeper, we don't need to append old entries, as we are fetching all entries every time.

Reply:
Done